### ISSUE TITLE
rename dataset to: Monitoring of invasive alien plant species by the Province East Flanders, Belgium

### ISSUE THREAD

**Issue by DimEvil** [riparias/pov-ias-plants-occurrences#80]
rename dataset to: Monitoring of invasive alien **plant** species by the Province East Flanders, Belgium

I think that this title would cover the content of the dataset better as there are no animals involved

**Comment by damianooldoni** [comment]
I agree. @LienReyserhove: do you agree too?

**Comment by sannegovaert** [comment]
I also agree, I just searched for this dataset on the IPT and needed different trials because I was searching for "plants".

**Comment by damianooldoni** [comment]
Thanks @sannegovaert for agreeing on this. Once we merge #113 to main, you can fix this. Thanks.

It's just a few steps:
- Modify `datasetName` in https://github.com/riparias/pov-ias-plants-occurrences/blob/513597adb524015a93739895981a29e6b8b3b308/sql/dwc_occurrence.sql#L13
- Run all chunks in `/src/dwc_mapping.Rmd`
- Commit
- Create PR, you can set me as reviewer

Many thanks.
